Loop/Managers/Alerts/AlertManager.swift

```diff
@@ -54,7 +54,9 @@ public final class AlertManager {
         self.userNotificationCenter = userNotificationCenter
         self.fileManager = fileManager
         let documentsDirectory = fileManager.urls(for: .documentDirectory, in: .userDomainMask).first
-        self.alertStore = alertStore ?? AlertStore(storageDirectoryURL: documentsDirectory, expireAfter: expireAfter)
+        let alertStoreDirectory = documentsDirectory?.appendingPathComponent("AlertStore")
```

Comment:
What happens to the overall functionality if `documentsDirectory` is nil? Would it better to just force unwrap it on the line above? That is, `.first!`?

Reply:
No.  This is intentional, because for Unit testing it could be nil